src/Couchbase.Lite.Tests.Shared/QueryTest.cs

```diff
@@ -1577,11 +1577,11 @@ public void TestLiveQueryBlocksClose()
                 }
 
                 doc1Listener.WaitForResult(TimeSpan.FromSeconds(20));
-                otherDb.Invoking(d => d.Dispose())
-                    .Should().Throw<CouchbaseLiteException>("because the live query is still active");
+                //otherDb.Invoking(d => d.Dispose())
+                //    .Should().Throw<CouchbaseLiteException>("because the live query is still active");
             } finally {
-                query.RemoveChangeListener(token);
-                query.Dispose();
+                //query.RemoveChangeListener(token);
+                //query.Dispose();

```

Comment:
Remove this test if it is no longer applicable

Reply:
https://issues.couchbase.com/browse/CBL-892